android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.java

```diff
@@ -238,7 +242,7 @@ public void onUpdate() {
     }
 
     // background
-    if (mBackgroundColor != 0) {
+    if (mBackgroundColor != -1) {
```

Comment:
Can you explain why this change is made here? If we expect "default" to be `-1` maybe we should also set the default in `@ReactProp` while defining bgColor:

```java
@ReactProp(name = "backgroundColor", customType = "Color", defaultInt = -1)
```

??

Reply:
I think it is a good idea to add that for extra safety.

Reply:
Sure. will take care of it as soon as possible.

There is still an open issue from @kyle-ssg which reproduces only if there is a custom left component. I won't have time earlier than next week to investigate

Reply:
But I think this is not an issue since it is the expected behavior for the title to disappear when there is `headerLeft` component.

Reply:
Regardless of if the headerLeft occurred before / after this work I think this is good to merge, I'll look into this separately and raise something if I think there's an issue. @WoLewicki @andreibarabas.

Reply:
On iOS, the situation is kind of similar but for `headerCenter`, where using it makes the title disappear.

Reply:
sure. it's fine with me. I added @kmagiera suggestion, so let me know if you need anything else from my side to get this merged.

Reply:
I am testing these changes out and this line seems to have introduced a bug where I can't set the header background color to "#fff" or "#ffffff".